shopify/assets/monitors/order_cancellation_rate.json

```diff
@@ -0,0 +1,74 @@
+{
+	"version": 2,
+	"created_at": "2024-10-01",
+	"last_updated_at": "2024-10-01",
+	"title": "Order Cancellation Rate is High",
+	"description": "High order cancellation rate is monitored.",
```

Comment:
Can you add more context, following the [best practices guide](https://docs.datadoghq.com/developers/integrations/create-an-integration-recommended-monitor/#description):

- This is not a copy of the title.
- Define the problem stated by the title.
- Answer why this is an issue worth alerting on.
- Describe the impact of the problem.

Reply:
Hi @estherk15, we have updated the description. please have a look.